tests/integration/docusaurus/deployment_patterns/databricks_deployment_patterns_dataframe_yaml_configs.py

```diff
@@ -142,7 +143,7 @@
 validator.expect_column_values_to_not_be_null(column="passenger_count")
 
 validator.expect_column_values_to_be_between(
-    column="congestion_surcharge", min_value=0, max_value=1000
```

Comment:
updated to use `Spark` DF directly. And adjusted the column to a better one (`congestion_surcharge` is always 0)